src/content/docs/style-guide/word-choice/nerdgraph-doc-guidelines.mdx

```diff
@@ -0,0 +1,32 @@
+* **NerdGraph explorer.** We mainly refer to this as the "NerdGraph explorer" but if it's in a context where it may not be obvious what exactly NerdGraph is, you can refer to the "NerdGraph API explorer." Other tips: 
+    * Avoid using just "explorer" because that conflicts with other explorers (e.g., the REST API explorer, or New Relic Explorer).
+    * We can explain, if we think it's helpful, that the NerdGraph explorer is built with GraphiQL, an open source platform for building GraphQL queries. This may be helpful to explain because both GraphQL and GraphiQL are known in the industry. 
+* **Requests.** There are two main types of GraphQL operations: queries and mutations ([defined here](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#terminology)). We can use these terms to refer to specific examples (for example: "here's a mutation that..."). To make a general reference to multiple NerdGraph requests of different kinds, you can use "requests." 
+* **Fields.** Fields are the elements on an object type that can be queried. For examples of "field" being used in context, see [this NerdGraph tutorial](/docs/nerdgraph-anomaly-detector-configurations-api-tutorial/#update-configuration). The reason fields are an important concept is that with GraphQL-format APIs, users specify exactly the fields they want back (in contract to REST APIs, for example, which can return a default response with fields you don't need). 
+
+## Tips for creating new NerdGraph docs [#creating-nerdgraph-docs]
+
+Tips for adding NerdGraph content to the public docs: 
+* When writing a new NerdGraph tutorial doc, look through the existing tutorial docs and choose one that is most similar to your content. Copy that and use it as a template. For more details about what kinds of information the different sections should have, see [the template](/docs/style-guide/writing-docs/article-templates/graphql-api-tutorial-template). 
+* When adding new NerdGraph content, check to see if you can include new content in existing NerdGraph docs. This helps us keep the list of tutorials clean and prevent it from sprawling. For example, if you want to add something related to viewing or configuring entities, you might add that to the [entity-related NerdGraph doc](/docs/apis/nerdgraph/examples/nerdgraph-entities-api-tutorial). 
+* When you add a new type of NerdGraph content, add it to the [index of NerdGraph content](/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph/#tutorials). That index was created to reflect that we have tutorials available that aren't in the main NerdGraph docs section.
+* To better format requests: if you want to "prettify" a request, there's a prettify button in the NerdGraph explorer. The request must be a working one, with real values. 
```

Comment:
@zuluecho9, you might clarify like this: If you want to format a request in NerdGraph explorer, click **Tools > Prettify** . The request must be a working one, with real values.